pkg/bindings/test/containers_test.go

```diff
@@ -755,8 +755,23 @@ var _ = Describe("Podman containers ", func() {
 		// Validate list container with id filter
 		filters := make(map[string][]string)
 		filters["id"] = []string{cid}
-		c, err := containers.List(bt.conn, filters, bindings.PTrue, nil, nil, nil, nil)
+		c, err := containers.List(bt.conn, filters, bindings.PTrue, nil, nil, nil)
 		Expect(err).To(BeNil())
 		Expect(len(c)).To(Equal(1))
 	})
+
+	It("List containers always includes pod information", func() {
```

Comment:
dont we need a test in the test/endpoint that would protect this?

Reply:
Do we? I think the bindings test captures it adequately - verifies it's in the response?